src/views/certifications/components/signup-form-area/components/newsletter-signup-form/marketo-form-data.module.css

```diff
@@ -0,0 +1,65 @@
+.formStyleOverrides {
+	display: grid;
+	width: 100%;
+	margin-top: 8px;
+	gap: 24px;
+
+	/* Modify email label text styles to match dev dot */
+	& [data-field-id='Email'] label {
+		font-size: var(--token-typography-body-200-font-size);
```

Comment:
I'd love for us to possibly collab on a set of shared design tokens so that the component can use one token that gets a different value for these based on global styles.

Reply:
Sweet, I had a vague but similar thought! Made a task here for our Dev Dot backlog: 🎟️ [[discuss] Approach to Marketo forms in Dev Dot](https://app.asana.com/0/1202097197789424/1203566191034158/f)